src/update_agent/mod.rs

```diff
@@ -238,6 +263,78 @@ impl UpdateAgent {
     }
 }
 
+/// Attempt to broadcast msg to all sessions registered in systemd's login manager.
+/// Returns a Result with a tuple of total sessions found and sessions broadcasted to,
+/// if no error.
+fn broadcast(msg: &str) -> Fallible<(u8, u8)> {
+    let sessions = get_user_sessions()?;
+    let sessions_total = sessions.len() as u8;
```

Comment:
Keeping everything as `usize` seems fine to me. Any specific reasons for casting to `u8` here instead?

Reply:
No specific reason. I noticed instances of `u8`s used elsewhere in Zincati and I thought smaller is probably better.
(Just read up a bit on it though; in this case it should probably be smarter to let the compiler choose for us).